Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -120,3 +120,7 @@
 _hgcal_RawToDigi += hgcalDigis
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toReplaceWith(RawToDigi,_hgcal_RawToDigi)
+
+_hgcal_RawToDigi += hfnoseDigis
+from Configuration.Eras.Modifier_phase2_hfnose_cff import phase2_hfnose
+phase2_hfnose.toReplaceWith(RawToDigi,_hgcal_RawToDigi)
```

Comment:
This code will add `hfnoseDigis` to `RawToDigi` regardless of `phaes2_hfnose` (as long as `phase2_hgcal is active; because line 122 "replaces" `RawToDigi` object with `_hgcal_RawToDigi`, and line 124 modifies `_hgcal_RawToDigi`, and therefore `RawToDigi` gets updated as well).

I suggest either
* add `copy()` to line 122, i.e. `phase2_hgcal.toReplaceWith(RawToDigi,_hgcal_RawToDigi.copy())`, or
* copy the sequence first before modifying
```python
_hfnose_RawToDigi = _RawToDigi.copy()
_hfnose_RawToDigi += hfnoseDigis
# etc
```
The copy, add, and replace could be made on a single line as well
```python
phase2_hfnose.toReplaceWith(RawToDigi, cms.Sequence(RawToDigi.copy()+hfnoseDigis))
```

Reply:
Done